pkg/sync/cluster.go

```diff
@@ -244,7 +244,22 @@ func launchWorker(address string, config *Config, wg *sync.WaitGroup) {
 				return
 			}
 			// launch itself
-			var args = []string{host, rpath}
+			var args = []string{host}
+			// set env
+			var printEnv []string
+			for k, v := range config.Env {
+				args = append(args, fmt.Sprintf("%s=%s", k, v))
```

Comment:
should we encode the `v`? for example, it may contain `&`

Reply:
use JFS='2&222 juicefs sync xxxx'  rather then `JFS=2&222 juicefs sync xxxxx`

<img width="2553" alt="image" src="https://user-images.githubusercontent.com/31313340/220045451-3b6f90be-85f4-4235-8c76-a0eba0d98642.png">

Reply:
yes